# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/CoalescedMouseData.h
# Commit: 5718f7f3067e
# Full Hash: 5718f7f3067eff42e097a6013acd82fbe6aa299a
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-05-15 22:58:00
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar
#   
#   This patch tries to dispatch `ePointerRawUpdate` with
#   `PresShell::EventHandler::DispatchPrecedingPointerEvent` as same as usual
#   pointer events.
# ==============================================================================

diff -r aae164cff6aa -r 5718f7f3067e dom/ipc/CoalescedMouseData.h
--- a/dom/ipc/CoalescedMouseData.h	Thu May 15 03:20:28 2025 +0000
+++ b/dom/ipc/CoalescedMouseData.h	Thu May 15 03:20:29 2025 +0000
@@ -27,7 +27,8 @@
 
   bool CanCoalesce(const WidgetMouseEvent& aEvent,
                    const ScrollableLayerGuid& aGuid,
-                   const uint64_t& aInputBlockId);
+                   const uint64_t& aInputBlockId,
+                   const nsRefreshDriver* aRefreshDriver);
 };
 
 class CoalescedMouseMoveFlusher final : public CoalescedInputFlusher {